src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -85,6 +85,81 @@ public override string ToString()
     }
 }";
 
+        [Fact, WorkItem(32495, "https://github.com/dotnet/roslyn/issues/32495")]
```

Comment:
Please add tests for:
When `f` is both not null and when it's defined in an oblivious context.
Doubly nested (Do we warn on both the nested `f2 = { ... }` and the outer `f = { f2 = { ... } }`?)
Object initializer inside an initializer that returns a nullable reference.

Reply:
I don't understand the last suggestion

---
In reply to: [266132886](https://github.com/dotnet/roslyn/pull/34141#discussion_r266132886) [](ancestors = 266132886)